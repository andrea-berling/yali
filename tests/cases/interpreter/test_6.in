// This program tests that leading whitespace are ignored while executing
print 72;
print 72 + 44;
print 72 + 44 + 54;
