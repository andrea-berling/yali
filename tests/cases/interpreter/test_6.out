72
116
170
